store/postgres/src/primary.rs

```diff
@@ -1247,18 +1247,33 @@ impl<'a> Connection<'a> {
     }
 
     /// Create a site for a brand new deployment.
+    /// If it already exists, return the existing site.
+    /// and a boolean indicating whether a new site was created.
+    /// false means the site already existed.
```

Comment:
Nice comment, just need to check ponctuation and capitalization.

Reply:
ah thanks for catching that. I'll fix that